haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku, \
+intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alexey Burshtein
+	2001-2002 Kevin H. Patterson"
+LICENSE="MIT
+	BSD (3-clause)"
+REVISION="1"
+srcGitRev="2fd70537847d3b9712e96ebbc4ff37d5e961b480"
+SOURCE_URI="https://github.com/HaikuArchives/Eventual/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="229cd7fa206da0606d32fc0db1f44a5eee20f0e8428e3dec141e4aab272ead68"
+SOURCE_FILENAME="Eventual-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Eventual-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
```

Comment:
Eventual builds and runs on x86_gcc2, but it fails to build on x86_64 with this error:
```
In file included from ActivityWindow.h:19:0,
                 from ActivityData.cpp:8:
../../Libraries/Category/Category.h:9:21: fatal error: ostream.h: No such file or directory
```
On x86_gcc2, `ostream.h` is in `/system/develop/headers/c++/2.95.3/`.
Does someone know if there is an easy patch that would make the app build on x86_gcc2?
Thanks!

Reply:
This patch allows to skip the missing ostream.h and go a few steps forward.
```
diff --git a/src/Libraries/Category/Category.h b/src/Libraries/Category/Category.h
index deaaef1..02032bd 100644
--- a/src/Libraries/Category/Category.h
+++ b/src/Libraries/Category/Category.h
@@ -6,7 +6,6 @@
 #define _CATEGORY_H_
 
 #include <iostream>
-#include <ostream.h>
 #include <stdlib.h>
 
 #include <InterfaceKit.h>
diff --git a/src/Libraries/Category/CategoryItem.h b/src/Libraries/Category/CategoryItem.h
index 118f4b4..4ba9779 100644
--- a/src/Libraries/Category/CategoryItem.h
+++ b/src/Libraries/Category/CategoryItem.h
@@ -6,7 +6,6 @@
 #define _CATEGORY_ITEM_H_
 
 #include <iostream>
-#include <ostream.h>
 #include <stdlib.h>
 
 #include <InterfaceKit.h>
-- 
2.16.0
```
But then we hit other issues (still on x86_64):
```
CalendarModulePreferences.cpp:42:1: warning: deprecated conversion from string constant to 'char*' [-Wwrite-strings]
CalendarModulePreferences.cpp: In constructor 'CalendarModulePreferences::CalendarModulePreferences(const BString&, BMessage*)':
CalendarModulePreferences.cpp:538:43: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
    ( this->weekends )->AddItem( ( void* ) tempUint32 );
```
So let's forget about x86_64. We can probably leave it as is (`?x86`).
(I'll delete this message and the previous one.)